server/views/formPages/openConditions/previousSentences.html

```diff
@@ -0,0 +1,93 @@
+{% extends "../formTemplate.html" %}
+{% from "radios/macro.njk" import govukRadios %}
+{% from "details/macro.njk" import govukDetails %}
+
+{% set greenButtonText = "Continue" %}
+
+{% set pageTitle = "Previous sentences" %}
+
+{% block navigation %}
+
+{% include "../../partials/breadCrumbTaskList.html" %}
+
+{% endblock %}
+
+{% set sevenOrMoreYearsTextHtml %}
```

Comment:
Suggest changing to releasedLastFiveYearsTextHtml

Reply:
done